readme.md

```diff
@@ -124,6 +124,72 @@ Also, you may be interested in adjusting Nextcloud's Datadir to store the files
 
 ⚠️ **Please note:** Almost all commands in this project's documentation use `sudo docker ...`. Since `sudo` is not available on Windows, you simply remove `sudo` from the commands and they should work.  
 
+
+### Synology DSM Setup
+<details>
+    <summary>For a AIO setup in Docker on a Synology NAS, there are a few adjustments necessary to make it run well, especially when running the integrated reverse proxy or firewall.</summary>
+    
+    
+The main issue is that installing AIO through the **Docker GUI doesn't work**, so you need to either use SSH or create a user-defined script task in the task scheduler as the user 'root' and run it.
+Before you start make sure to create two folders on you NAS `/volume1/docker/nextcloud/data` `/volume1/docker/nextcloud/backup`
```

Comment:
That should not be necessary as the folders should get created automatically if they are not present.

```suggestion
```